### ISSUE TITLE
fix(email-access) resolve jquery error on email access form

### ISSUE THREAD

**Issue by Benunc** [WordImpress/Give#3561]
## User Story

As a `donor`, I want `to be able to use the email access feature to access my donation history and/or subscription history`.

## Current Behavior
I currently enter my email in the email access form and the button is not clickable. 

In the console I see this error:

```
form.js:376 Uncaught TypeError: Cannot read property 'give_form_hash' of undefined
    at Object.success (form.js:376)
    at i (jquery.js?ver=1.12.4:2)
    at Object.fireWith [as resolveWith] (jquery.js?ver=1.12.4:2)
    at y (jquery.js?ver=1.12.4:4)
    at XMLHttpRequest.c (jquery.js?ver=1.12.4:4)
```

## Expected Behavior

I expect the form to work.

## Steps to Reproduce
1. On Give 2.2, enable email access.
2. in a logged out browser, navigate to the donation-history page.
3. Attempt to enter an email.

Actually, before step 3 you can already see the error, even before entering anything.

## Related
- https://secure.helpscout.net/conversation/633566496/22862?folderId=848135
- https://secure.helpscout.net/conversation/633581205/22863/?folderId=672194

## Tasks
- [ ] fix the error on the donation history page
- [ ] ensure that the fix works on the subscriptions page with recurring as well

## Environment

<details>
    <summary>WordPress System Info</summary>
### WordPress Environment ###

Home URL: https://livegive.wpsteward.com
Site URL: https://livegive.wpsteward.com
WP Version: 4.9.8
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: wp_8138bfdc07_
Table Prefix Length: 14
Table Prefix Status: Acceptable
Admin AJAX: Accessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, request-pending, request-confirmed, request-failed, request-completed, refunded, failed, revoked, cancelled, abandoned, processing, preapproval, give_subscription

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: livegive.wpsteward.com
TLS Connection: Connection uses TLS 1.2
TLS Connection: Probably Okay
Server Info: Apache/2.4.33 (Unix) OpenSSL/1.0.1f
PHP Version: 7.0.31
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
PHP Max Upload Size: 512 MB
cURL Version: ❌ 7.35.0, OpenSSL/1.0.1f - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.61 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 2.2.1
Give Cache: Enabled
Database Updates: All DB Updates Completed.
Give Cache: Enabled
Give Cache: ✔New Donation✔Donation Receipt❌New Offline Donation❌Offline Donation Instructions✔New User Registration✔User Registration Information✔Email access❌Renewal Receipt Email❌Subscription Cancelled Email❌Subscription Completed Email❌Subscription Reminder Email✔Subscriptions Email Access
Upgraded From: –
Test Mode: Enabled
Currency Code: USD
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: https://livegive.wpsteward.com/donation-confirmation/
Failure Page: https://livegive.wpsteward.com/transaction-failed/
Donation History Page: https://livegive.wpsteward.com/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, Stripe - Credit Card
Default Payment Gateway: Test Donation
PayPal IPN Verification: Enabled
PayPal IPN Notifications: IPN received for #585 ( 7FN59381E35027022 ) on 05/30/2018 at 15:13. Status VERIFIED
Donor Email Access: Enabled

### Active Give Add-ons ###

Give - Authorize.net Gateway: ✔ Licensed – by WordImpress – 1.4.2
Give - Fee Recovery: ✔ Licensed – by WordImpress – 1.7.1
Give - Google Analytics Donation Tracking: ✔ Licensed – by WordImpress – 1.2.0
Give - Manual Donations: ✔ Licensed – by WordImpress – 1.4.1
Give - PayPal Pro Gateway: ✔ Licensed – by WordImpress – 1.1.6
Give - PDF Receipts: ✔ Licensed – by WordImpress – 2.3
Give - Recurring Donations: ✔ Licensed – by WordImpress – 1.7.2
Give - Tributes: ✔ Licensed – by WordImpress – 1.5.0
Give - Zapier: ✔ Licensed – by WordImpress – 1.2.1

### Other Active Plugins ###

Give - Stripe Gateway: by WordImpress – 2.0.9
Google Analytics for WordPress by MonsterInsights: by MonsterInsights – 7.0.9
User Switching: by John Blackbourn & contributors – 1.3.1
WP DSGVO Tools: by Shapepress eU – 1.5.6

### Inactive Plugins ###

Akismet: by Automattic – 3.1.11
Email Cop: by Ashfame – 0.1.1
Give - Database HealthCheck: by WordImpress – 0.0.2
Give - Mollie Gateway: by WordImpress – 1.1.1
Hello Dolly: by Matt Mullenweg – 1.6
RegistrationMagic Premium: by Registrationmagic – 3.8.1.7
Restrict Content Pro: by Restrict Content Pro Team – 2.9.10
Series Engine: by Eric Murrell (Volacious) – 2.6.1
WordPress Importer: by wordpressdotorg – 0.6.3

### Active MU Plugins ###

api-hacks.php: by  –

### Theme ###

Name: Twenty Seventeen
Version: 1.6
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme
</details>

**Comment by kevinwhoffman** [comment]
@raftaar1191 Please bump this to the top of your list.

**Comment by ravinderk** [comment]
@Benunc @kevinwhoffman This is already fixed in `release/2.2.2`: https://github.com/WordImpress/Give/issues/3553
